conduct.md

```diff
@@ -106,6 +94,23 @@ listed above,
   * Sébastien Doeraene, Scala Center ([sebastien.doeraene@epfl.ch](mailto:sebastien.doeraene@epfl.ch))
   * Kathleen Hayes, Lightbend ([kathleen.hayes@lightbend.com](mailto:kathleen.hayes@lightbend.com))
 
+### Examples of harassment
+
+The term “Harassment” includes “Unacceptable Behavior” described in the [Citizen Code of Conduct](http://citizencodeofconduct.org/).
```

Comment:
What's the reason for reverting the changes in https://github.com/scala/scala-lang/commit/2f76f5246deff5c7423b8cf16c15a9757c420c6b? The purpose was to simplify the main text of this CoC not to depend on another document.

Reply:
The reason is to avoid the possible interpretation that these examples were taken from historical events in our community. Since these examples were taken (verbatim) from the Citizen Code of Conduct it would be good to keep the reference and clarify that they are from that resource.

Reply:
Note also that it's not just a reversion. The bullets are re-added later in the document.

Reply:
I remain thoroughly opposed to defining harassment as including those bulleted points for no apparent benefit. It doesn't matter whether those things are included in our definition of harassment. What matters is that those things are unacceptable, regardless of the term "Harassment" does or doesn't include them.

Reply:
This is a good point, @martijnhoekstra: “harassement” is a legal term already, and its definition does not match these bullet points in a legal system I know, so should we change the title to “Unacceptable Behavior”?

Reply:
changed to "Examples of unacceptable behavior"